### DISCUSSION TITLE
139

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#73] [General]
# 潮流周刊第139期 - 吃饭家伙

封面图来源于上周五领到的新电脑 M2 Pro 16 寸，周末把玩了两天，很是舒服，速度相比之前那个 16，要快太多了，电量很惊人，键盘很舒服，大爱。 

https://weekly.tw93.fun/posts/139-%E5%90%83%E9%A5%AD%E5%AE%B6%E4%BC%99

**Comment by zimiyu** [comment]
老高和小茉 主要是捧哏的捧的好，冷不丁插句嘴，能笑岔气

**Reply by tw93** [reply]
哈哈 当做故事会看

**Comment by bravesfly** [comment]
LocationSimulator的地址错误，应为https://github.com/Schlaubischlump/LocationSimulator

**Reply by tw93** [reply]
谢谢提醒 我修改一下
